docs/static/docker.asciidoc

```diff
@@ -3,8 +3,16 @@
 Docker images for Logstash are available from the Elastic Docker
 registry.
 
-Obtaining Logstash for Docker is as simple as issuing a +docker pull+
-command against the Elastic Docker registry.
+The base image is https://hub.docker.com/_/centos/[Centos 7] and the source
```

Comment:
Let's try to stay consistent with the name here across all Docker docs for products. I used `centos:7` in Elasticsearch as this is the unambiguous way to reference the base image in Docker Hub. If we stick to this, the proper capitalization is `CentOS 7`.